src/tests/login.test.js

```diff
@@ -0,0 +1,145 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import models from '../models';
+import app from '../index';
+import userFixtures from '../fixtures/user-fixtures';
```

Comment:
`<'../fixtures/user-fixtures'>` must be changed to `'../fixtures/user.fixtures'`